components/legacy_sidebar/header/sidebar_header_dropdown_button.jsx

```diff
@@ -66,6 +76,22 @@ export default class SidebarHeaderDropdownButton extends React.PureComponent {
                         className='user__name'
                     >
                         {'@' + this.props.currentUser.username}
+                        {statusEmoji &&
```

Comment:
We shouldn't need the whole Markdown component here to render the emoji. I think we should have something simpler, and if it is not there, we should create it.

Reply:
I checked codebase but didn't find any reference of rendering emoji without markdown component. I found some old commits where emoji rendering was added for some components and they used markdown so I did the same.

Reply:
@darkLord19 Maybe these utils would be useful?
- https://github.com/mattermost/mattermost-webapp/blob/371c302e143727ce6e2c93383e48815ea7d7ec16/utils/text_formatting.tsx#L839
- https://github.com/mattermost/mattermost-webapp/blob/371c302e143727ce6e2c93383e48815ea7d7ec16/utils/emoticons.tsx#L90